truncate_table_src/truncate_table.py

```diff
@@ -0,0 +1,37 @@
+import os
+import logging
+import boto3
+
+rds_client = boto3.client('rds-data')
+
+# Initialise logging
+logger = logging.getLogger(__name__)
+log_level = os.environ["LOG_LEVEL"] if "LOG_LEVEL" in os.environ else "INFO"
+logger.setLevel(logging.getLevelName(log_level.upper()))
+logger.info("Logging at {} level".format(log_level.upper()))
+
+RDS_CREDENTIALS_SECRET_ARN = os.environ["RDS_CREDENTIALS_SECRET_ARN"]
+RDS_DATABASE_NAME = os.environ["RDS_DATABASE_NAME"]
+RDS_CLUSTER_ARN = os.environ["RDS_CLUSTER_ARN"]
+
+
+def execute_statement(sql, database_name=None):
+    args = {
+        "secretArn": RDS_CREDENTIALS_SECRET_ARN,
+        "resourceArn": RDS_CLUSTER_ARN,
+        "sql": sql
+    }
+    if database_name is not None:
+        args["database"] = database_name
+
+    response = rds_client.execute_statement(**args)
```

Comment:
`**kwargs ` would be clearer here as it is expanding them to `<key word>=<value>` and not just args.